action.yml

```diff
@@ -20,6 +20,18 @@ inputs:
     required: false
     default: 'all'
     description: 'The type of tests to be run by the test runner.'
+  enableCodeCoverage:
+    required: false
+    default: 'false'
+    description: 'Should code coverage results be collected for the test run.'
+  coverageAssemblyFilters:
+    required: false
+    default: ''
+    description: 'Optional coverage assembly filters when collecting code coverage.'
```

Comment:
Would it be possible to come up with sensible defaults instead of exposing this in our public api in the first iteration?

Reply:
It would probably be best to include an option for `-coverageOptions` with `enableCyclomaticComplexity;generateHtmlReport;generateBadgeReport` as the default argument. And an end user could add in `assemblyFilters:+MyAssembly` if needed. Makes much more sense to include it this way.

The only reason this would become a bit difficult is that combining results from playmode and test mode can be quite difficult, but I'll keep looking into it for now.